packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1606,8 +1606,6 @@ export class ContainerRuntime
 				!readonly || !this.connected,
 				0x125 /* "Unsafe to transition to read-only state!" */,
 			);
-
-			this.replayPendingStates();
```

Comment:
Why is this removed? "readonly" doesn't seem to have any related to this bug and the scenario in the PR description.